standard/bridge-v1/pkg/transfer/transfer.go

```diff
@@ -108,16 +118,22 @@ func NewTransferToL1(
 	}
 
 	return &Transfer{
-		amount:        amount,
-		destAddress:   destAddress,
-		privateKey:    privateKey,
-		srcClient:     commonSetup.settlementClient,
+		amount:      amount,
+		destAddress: destAddress,
+		privateKey:  privateKey,
+		srcClient: shared.NewETHClient(
+			logger.With("component", "l1_eth_client"),
+			commonSetup.l1Client,
+		),
 		srcChainID:    commonSetup.settlementChainID,
 		srcTransactor: st,
 		srcFilterer:   sf,
-		destClient:    commonSetup.l1Client,
-		destFilterer:  l1f,
-		destChainID:   commonSetup.l1ChainID,
+		destClient: shared.NewETHClient(
+			logger.With("component", "settlement_eth_client"),
+			commonSetup.settlementClient,
```

Comment:
Dest should use l1Client as before
```suggestion
			commonSetup.l1Client,
```